src/main/java/com/rettichlp/unicacityaddon/events/faction/terroristen/BombTimerEventHandler.java

```diff
@@ -30,6 +27,7 @@
 public class BombTimerEventHandler {
 
     private static String location;
+    public static int activeBomb = 0;
```

Comment:
Hast du das mal getestet? Weil theoretisch müsste so wie es jetzt ist bei jedem Spielstart (15 Ticks nach dem Spielstart) ein Screenshot gemacht werden, da im TickEventHandler bei handleBombScreenshot() du größer-GLEICH checkst...

Wenn du das hier auf -1 statt 0 setzt würde das verhindert werden:
```suggestion
    public static int activeBomb = -1;
```